app/src/main/java/com/heartoracle/sport/student/core/utils/Utils.kt

```diff
@@ -0,0 +1,25 @@
+package com.heartoracle.sport.student.core.utils
+
+
+import kotlin.collections.ArrayList
+
+
+fun ArrayList<Int>.getMode(): Int {
+    var count = 1
+    var tempCount: Int
+    var popular = this[0]
+    var temp = 0
+    for (i in 0 until this.size - 1) {
```

Comment:
когда until не надо минусовать 1

Reply:
и почему не foreach?

Reply:
```suggestion
    for (i in 0 until this.size - 1) {
```потому что код сп*здил

Reply:
> и почему не foreach?

потому что код сп*здил